src/main/kotlin/com/bada/app/models/Item.kt

```diff
@@ -13,7 +13,7 @@ class Item(
     val name: String,
     val description: String,
 
-    @ManyToMany(mappedBy = "items")
+    @ManyToMany
     val warehouses: List<Warehouse>,
```

Comment:
Map the `WarehouseItem` instead, so we can display stock

Reply:
Fixed